src/js/components/charts/ChartStripes.js

```diff
@@ -15,7 +14,7 @@ var ChartStripes = React.createClass({
     var count = props.count;
     var width = props.width / (2 * count);
 
-    return _.map(_.range(0, count), function (i) {
+    return Object.keys(Array(count).fill()).map(function (i) {
```

Comment:
We'll need `Array(count).fill().map()` because without `.fill()` it would be `Array [ <3 empty slots> ]` but yes that would be better, thanks! :D